app/src/androidTest/java/com/microsoft/device/samples/dualscreenexperience/presentation/launch/LaunchDualScreenTest.kt

```diff
@@ -71,6 +71,21 @@ class LaunchDualScreenTest {
         checkLaunchInDualMode()
     }
 
+    @Test
+    fun openMainInDualPortraitModeWithRotations() {
+        device.spanFromStart()
+
+        checkDualLaunchButton()
+        clickDualLaunchButton()
+
+        checkMapFragment()
+        device.setOrientationRight()
```

Comment:
Why do you need both of them?

Reply:
We discussed and also added checks after the orientations to ensure we cover more cases.